src/security/ServerOptions.cc

```diff
@@ -413,8 +419,6 @@ Security::ServerOptions::loadDhParams()
             if (OSSL_DECODER_from_fp(dctx.get(), in)) {
                 assert(rawPkey);
                 const Security::DhePointer pkey(rawPkey);
-                // TODO: verify that the loaded parameters match the curve named in eecdhCurve
-
```

Comment:
This is still valid. Not all EC curves accept the same parameter values, and some RFC 4492 section 5.1.1 specifies "explicit-prime" curves which have no name but are determined by their parameters.

Reply:
AFAIK, at least at the current moment in time, at least openssl does not support the generation of any arbitrary EC parameters. Neither do I see any support for 
```
          arbitrary_explicit_prime_curves(0xFF01),
          arbitrary_explicit_char2_curves(0xFF02),
``` 
in `openssl ecparam -list_curves` (which I know is not the final truth but probably a good indication of how widespread those are in actual use)